RedHatInsights/compliance-frontend
# fix(EditPolicy): RHICOMPL-1783 - Correct behaviour of alert for new rules

This corrects the behaviour of the alert about new rules at the bottom of the "Systems" tab.
it now correctly identifies if there is a new set of rules to configure if a systems with a minor version is selected that had not yet been covered by a rule set.

*How to test:*

1. Find a profile with multiple supported RHEL minor versions
2. Select systems only of one minor version and create the policy
3. Edit the policy.
4. The modal on the Systems tab should not show the alert when first loaded.
5. Select systems of a different minor version
6. The Alert should appear and allow to switch to the "Rules" tab via the link
7. (When removing all systems of a specific minor version, the alert should disappear again)
8. After saving and reopening there should again not be any alert on the Systems tab


## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
